.gitattributes

```diff
@@ -0,0 +1,2 @@
+# do not expand these files by default in diff viewer
+go.sum linguist-generated
```

Comment:
I didn't know you could configure this in github and is a nice quality of life improvement 👍